jitsi_meet/handlers/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- name: jitsi_meet_update_repo_cache
+  apt:
+    update_cache: 'yes'
```

Comment:
Don't update the cache with a handler. Do it while `apt_repository` with the [update_cache](http://docs.ansible.com/ansible/latest/apt_repository_module.html) parameter.